Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = """
 This crate contains the physical engine of
 the simulation, as well as the objects that reside
 within it"""
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
Shouldn't version bumps be made in a separate PR?

Reply:
That's how I did it the [https://github.com/myelin-ai/engine/pull/65](last) few [times](https://github.com/myelin-ai/engine/pull/65) in accordance with @jeremystucki